packages/web-components/src/component-mixins/cta/video.ts

```diff
@@ -132,49 +132,51 @@ const VideoCTAMixin = <T extends Constructor<HTMLElement>>(Base: T) => {
       // Declaring this mixin as it extends `LitElement` seems to cause a TS error
       // @ts-ignore
       super.updated(changedProperties);
-      const { ctaType, videoName, videoDescription, href, videoDuration } =
-        this;
-      const { eventRequestVideoData } = this
-        .constructor as typeof VideoCTAMixinImpl;
-      if (changedProperties.has('ctaType') && ctaType === CTA_TYPE.VIDEO) {
-        if (typeof videoDuration === 'undefined') {
+      customElements.whenDefined(`${ddsPrefix}-text-cta`).then((response) => {
```

Comment:
I'm pretty sure that this should be targetting the `dds-video-cta-container`, should it not? 

My reasonsing is as follows. This method is dispatching an event named `dds-cta-request-video-data`. That event is [handled by `DDSVideoCTAComposite`](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1/packages/web-components/src/components/cta/video-cta-composite.ts#L88-L119). The relevant element in use on the page is `dds-video-cta-container`, (it extends from `DDSVideoCTAComposite`). Therefore, we should be targetting `dds-video-cta-container`.

Reply:
Also note that there is a Typescript error here. In the `.then()`, you’re declaring a `response` parameter that you never actually use. Just need to remove that `response` parameter, and go with an empty parameter list.

Reply:
@m4olivei, Thanks for the review!!.
Yes, yes it should. 
Addressed on my latest commit.